lib/batcher/options.go

```diff
@@ -64,10 +63,5 @@ default based on the batch_mode in use.
 `, opt.DefaultTimeoutAsync, opt.DefaultTimeoutSync),
 		Default:  fs.Duration(0),
 		Advanced: true,
-	}, {
-		Name:     "batch_commit_timeout",
-		Help:     `Max time to wait for a batch to finish committing`,
-		Default:  fs.Duration(10 * time.Minute),
-		Advanced: true,
```

Comment:
We don't normally remove options as it can break peoples setups.

We can hide them from the docs and configurator like this

https://github.com/rclone/rclone/blob/84cebb6872cd8786e86357407f9159ec4386ba85/backend/azureblob/azureblob.go#L369-L375

Reply:
Thanks, that makes sense. I tried to apply this approach. PTAL.